app/src/main/java/io/github/sds100/keymapper/mappings/keymaps/trigger/KeyMapTrigger.kt

```diff
@@ -165,3 +166,54 @@ object KeymapTriggerEntityMapper {
         )
     }
 }
+
+// Comparator.reversed() requires API level 24
+class KeyMapTriggerComparator(
+    private val reverse: Boolean = false,
```

Comment:
There should be a Kotlin way to do reverse comparators instead of having to add it manually. Is this useful? https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.comparisons/reverse-order.html

Reply:
I tried to find a Kotlin way and only suitable method was https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.comparisons/reversed.html but it requires at least android 7